src/datasource/components/FieldValueFrequency.tsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { Field } from 'datasource/types';
 import { Toggletip } from './Toggletip';
-import { HorizontalGroup, VerticalGroup } from '@grafana/ui';
+import { HorizontalGroup, VerticalGroup, Button } from '@grafana/ui';
 
 interface Props {
   field: Field;
   children: JSX.Element;
+  onPlusClick?: (field: Field, value: string) => void;
```

Comment:
Nit: this would probably be better named around the behavior instead of the icon - ie, `addAndClause`, `addNotClause` or similar.

Reply:
Discussed via Slack, this is fine.